task12/index.js

```diff
@@ -0,0 +1,56 @@
+const parent = document.getElementById("js-parent");
+    
+const appendButtonForParent = () => {
+    const button = document.createElement('input');
+    button.id = "button";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+appendButtonForParent();
+
+const removeButton = () => {
+    document.getElementById("button").remove();
+};
+
+const addEvent = document.getElementById("button"); 
+addEvent.addEventListener('click', () => {
+    removeButton();
+    init();
+});
```

Comment:
IMO
I think you can get through with the button.
```suggestion
const button = document.getElementById("button"); 
button.addEventListener('click', () => {
    removeButton();
    init();
});
```

Reply:
Thank you for pointing out the part that was supposed to care.
I was pointed out and realized that I didn't need a function to remove it.
I've been doing the same thing over and over again, so be careful. I learned a lot!